recipes/cothread/0001-epicscorelibs.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/cothread/__init__.py b/cothread/__init__.py
+index e0a8018..f67ff27 100644
+--- a/cothread/__init__.py
++++ b/cothread/__init__.py
+@@ -37,6 +37,10 @@ modules:
+     catools
+         Channel access support tools for client access to EPICS records.
+ '''
++try:
++    import epicscorelibs.path.cothread
++except ImportError:
++    pass
```

Comment:
Why is this needed?

Reply:
@jakirkham this is needed to use the runtime dependency to libca provided by epicscorelibs.
Users can also have this library at some externally built location on their system so that is why it is optional and wrapped with the try except

Reply:
Why not make a Conda package for that library as well?

Reply:
It is already on conda-forge... epicscorelibs.

Reply:
Then please use this as a run requirements. Conda packages should depend on all things that are required to run. Using libraries from somewhere else is not the default usage and should be avoided at all costs.

Reply:
@xhochy It is there already. https://github.com/conda-forge/staged-recipes/pull/9973/files#diff-0a6a7eb69a560925ee867b776505534aR30

Reply:
Why is this patch then still needed? (As you see if this is really needed, add a comment about this in the recipe so other people not ask again ;) )

Reply:
@xhochy this patch is needed because the python library looks at an environment variable for the location of the libca library. epicscorelibs provides this library and a wrapper for `cothread` to bypass this check. I will add a comment as suggested.
We use the same approach at other EPICS client libraries such as `pyepics` and `pyca`, all here at conda-forge.